README.md

```diff
@@ -1,6 +1,25 @@
 # readout - Readout software and utilities 
 Appfwk DAQModules, utilities, and scripts for DUNE Upstream DAQ Readout Software.
 
+## TP Integration
```

Comment:
Add TP integration to the END of the Readme, don't start with it. First the user needs to build it.. 
Also, the instructions are not clear. This is the readout package. We are not testing the TP readout, BUT the whole readout. So I would keep the Readme as it was, and on the end, add a small subsection with a link similar to the binary file link for the WIBs, and to activate the TP, create a different JSON file. Like: fakereadout-withtp-commands.json that contains the TP related config.